src/group_theory/nilpotent.lean

```diff
@@ -181,6 +181,15 @@ lemma upper_central_series_is_ascending_central_series :
   is_ascending_central_series (upper_central_series G) :=
 ⟨rfl, λ x n h, h⟩
 
+lemma upper_central_series_mono : monotone (upper_central_series G) :=
```

Comment:
Is the following more general result true?
```lean
lemma is_ascending_central_series.mono {H : ℕ → subgroup G} (h : is_ascending_central_series H) : monotone H :=
```

Reply:
i'm not sure.. i initially thought so but have yet to come up with a proof. perhaps best to remove this lemma from the PR until i know whether the more general case holds and if so have a proof for it?

Reply:
yeah i _think_ the definition of ascending and descending isn't strong enough to prove mono for all asc/desc series

Reply:
No, it's definitely better to have the less general version than no version at all - don't remove this unless you have the replacement ready!

Reply:
i have an equivalent lemma for lower central series ill PR in a few days probably and the same problem - no proof of the general case for all descending series. seems likely that this doesn't hold for all cases without extra conditions